OpenSim/Actuators/RigidTendonMuscle.cpp

```diff
@@ -193,8 +193,6 @@ calcMuscleDynamicsInfo(const State& s, MuscleDynamicsInfo& mdi) const
     mdi.fiberActivePower = -(mdi.activeFiberForce) * fvi.fiberVelocity;
     mdi.fiberPassivePower = -(mdi.passiveFiberForce) * fvi.fiberVelocity;
     mdi.tendonPower = 0;
-    mdi.musclePower = -getMaxIsometricForce()*mdi.normTendonForce
```

Comment:
Isn't this calculation different from the generic `Muscle` one?

Reply:
Overlooked that one... Looks like it was originally incorrect. @tkuchida what do you think?

Reply:
As a note: This pr doesnt change the computation. The fiberVelocity for the RigidTendonMuscle is equal to lengtheningSpeed. See https://github.com/opensim-org/opensim-core/blob/20752f3a7852854a3e95a0bf52832bc8ec788163/OpenSim/Actuators/RigidTendonMuscle.cpp#L164 
So it matches the computation in `Muscle`.
As to why fibervelocity equals muscle velocity, I am unsure :)

Reply:
Looks like a bug 🐞 to me, but perhaps I'm misunderstanding something. If the tendon is rigid, won't the lengthening speed of the path equal the velocity of the fiber only if pennation is zero? Unfortunately, RigidTendonMuscle is only tested when pennation is zero (see [`testRigidTendonMuscle()` in testMuscles.cpp](https://github.com/opensim-org/opensim-core/blob/20752f3a7852854a3e95a0bf52832bc8ec788163/OpenSim/Actuators/Test/testMuscles.cpp#L511)). In general,

$v^{\mathrm{M}}(t) = \frac{1}{\cos(\phi(t))} \left[ v^{\mathrm{MT}}(t) + \ell^{\mathrm{M}}(t) \sin(\phi(t)) \dot{\phi}(t) \right]$

which simplifies to $v^{\mathrm{M}}(t) = v^{\mathrm{MT}}(t)$ when $\phi(t) = 0$.

Reply:
It should not be assumed that pennation is zero.